template/package.json

```diff
@@ -18,7 +18,15 @@
     "reverse": "adb reverse tcp:8081 tcp:8081",
     "generateimages": "node scripts/genimg src/assets/images --output src/themes/image.js --name Images",
     "generatefonts": "node scripts/fonts",
-    "plop": "plop"
+    "plop": "plop",
+    "build-local": "cd ios && fastlane build type:build env:local",
+    "build-staging": "cd ios && fastlane build type:build env:staging",
+    "build-preprod": "cd ios && fastlane build type:build env:preprod",
+    "build-production": "cd ios && fastlane build type:build env:production",
+    "push-local": "bash ./scripts/codepush.sh Local",
+    "push-staging": "bash ./scripts/codepush.sh Staging",
+    "push-preprod": "bash ./scripts/codepush.sh PreProd",
+    "push-prod": "bash ./scripts/codepush.sh Production"
```

Comment:
Could you make the script shorter by ask use input env when they run the script.

Reply:
Could you give me an example?